example/spec/parallel/dbRecords/DbRecordsDLQSpec.js

```diff
@@ -1,93 +1,93 @@
 'use strict';
 
-const { fakeFileFactory, fakeGranuleFactoryV2 } = require('@cumulus/api/lib/testUtils');
-const { SQS } = require('@cumulus/aws-client');
+const { lambda, s3 } = require('@cumulus/aws-client/services');
 const { randomString } = require('@cumulus/common/test-utils');
 const {
   deleteS3Object,
   listS3ObjectsV2,
+  getObject,
+  getObjectStreamContents,
 } = require('@cumulus/aws-client/S3');
 const { waitForListObjectsV2ResultCount } = require('@cumulus/integration-tests');
 
 const { loadConfig } = require('../../helpers/testUtils');
-
-describe('When a bad record is sent on the DLQ', () => {
-  let beforeAllSucceeded = false;
+describe('when a bad record is ingested', () => {
   let stackName;
   let systemBucket;
-  let dbRecordsDLQUrl;
   let executionName;
   let failedMessageS3Key;
 
+  let beforeAllSucceeded = false;
   beforeAll(async () => {
     const config = await loadConfig();
     stackName = config.stackName;
-    systemBucket = config.bucket;
-
-    const DLQName = `${stackName}-sfEventSqsToDbRecordsDeadLetterQueue`;
-    dbRecordsDLQUrl = await SQS.getQueueUrlByName(DLQName);
-
-    const granuleId = randomString(10);
-    const files = [fakeFileFactory()];
-    const granule = fakeGranuleFactoryV2({ files, granuleId, published: false });
-
     executionName = `execution-${randomString(16)}`;
-
-    const failingMessage = {
-      cumulus_meta: {
-        workflow_start_time: 122,
-        cumulus_version: '8.0.0',
-        state_machine: 'arn:aws:states:us-east-1:1234:execution:state-machine-name:execution-name',
-        execution_name: executionName,
-      },
-      meta: {
-        status: 'failed',
-        collection: 'bad-collection',
-        provider: 'fake-provider',
-      },
-      payload: {
-        granules: [granule],
-      },
-    };
-
-    // Send the message directly on the DLQ. Sending the message on the input queue results in an
-    // extremely long-duration, unreliable test (>20 mins) because updates to the redrive policy
-    // are very slow and unreliable and our normal visibilityTimeout and maxReceiveCount are high.
-    await SQS.sendSQSMessage(dbRecordsDLQUrl, JSON.stringify(failingMessage));
-    beforeAllSucceeded = true;
+    systemBucket = config.bucket;
+    const { $metadata } = await lambda().invoke({
+      FunctionName: `${stackName}-sfEventSqsToDbRecords`,
+      InvocationType: 'RequestResponse',
+      Payload: JSON.stringify({
+        env: {},
+        Records: [{
+          Body: JSON.stringify({
+            detail: {
+              status: 'RUNNING',
+              input: JSON.stringify({
+                cumulus_meta: {
+                  execution_name: executionName,
+                },
+                a: 'sldkj',
+              }),
+            },
+          }),
+        }],
+      }),
+    });
+    if ($metadata.httpStatusCode < 400) {
```

Comment:
is any code under 400 really ok, or are you actually looking for 20x?

Reply:
I'm trying to be careful to get errors in the actual tests not here wherever possible so I've made as permissive as reasonable.
but if you think it's a better idea to look for 20x I'm open to it

Reply:
I'm not too concerned, as I don't know how likely it is, but seems like 10x and 30x would be accepted here. If you're trying to make sure it's not an error code then that makes sense.

Reply:
exactly, I'm trying to be lenient and just make sure it's not an error code